schemas/rules/boolean_rule.json

```diff
@@ -1,7 +1,7 @@
 {
-  "$id": "https://eq.ons.gov.uk/rules/rule.json",
+  "$id": "https://eq.ons.gov.uk/rules/boolean_rule.json",
```

Comment:
You're right, all our rules are boolean and this is explicit, but because we will never have a rule that isn't I think could have skipped these changes and reduced the noise